cypress/features/regression/test/actionPopoverEvents.feature

```diff
@@ -3,9 +3,9 @@ Feature: Action Popover component
 
   Background: Open Action Popover component page
```

Comment:
Please change the name of file `actionPopoverInIFrame`

Reply:
updated